src/components/content-tab-assistant.tsx

```diff
@@ -151,11 +151,17 @@ export const Message = ( { children, isUser, className }: MessageProps ) => {
 	};
 
 	return (
-		<div className={ cx( 'flex mt-4', isUser ? 'justify-end' : 'justify-start', className ) }>
+		<div
+			className={ cx(
+				'flex mt-4',
+				isUser ? 'justify-end md:ml-24' : 'justify-start md:mr-24',
```

Comment:
Drive-by fix, to add the left/right margins to messages so they appear to the appropriate side of the chat history.

<img width="666" alt="Screenshot 2024-06-12 at 11 33 29" src="https://github.com/Automattic/studio/assets/417538/71aa504b-f998-433f-a372-8c9886d6c1f6">